homeassistant/components/sensor/dte_energy_bridge.py

```diff
@@ -0,0 +1,82 @@
+"""Support for monitoring energy usage using the DTE energy bridge."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:flash'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the DTE energy bridge sensor."""
+    ip_address = config.get('ip')
+    if not ip_address:
+        _LOGGER.error(
+            "Configuration Error"
+            "'ip' of the DTE energy bridge is required")
+        return None
+    dev = []
+    dev.append(
+        DteEnergyBridgeSensor(ip_address,
+                              "Current Energy Usage", "instantaneousdemand",
```

Comment:
Originally I had another endpoint (Energy Summation).  I ended up taking it out because I couldn't think of a way to make sense of the data with HA.  (It's some arbitrarily high value that counts how many kWh was used)

Probably should take all this abstraction out until we actually need it.

- [ ] Remove unnecessary constructor arguments